packages/vue/test/errorHandler.test.ts

```diff
@@ -0,0 +1,425 @@
+import { getCurrentHub } from '@sentry/browser';
+
+import { generateComponentTrace } from '../src/components';
+import { attachErrorHandler } from '../src/errorhandler';
+import { Operation, Options, ViewModel, Vue } from '../src/types';
+
+const CURRENT_NODE_VERSION = process.version.replace('v', '').split('.')[0];
+const describeSkipNode8 = CURRENT_NODE_VERSION === '8' ? xdescribe : describe;
+
+describe('attachErrorHandler', () => {
+  describeSkipNode8('attachProps', () => {
```

Comment:
No I have no idea why they are failing. I don't know if it has to do with `jest` or the fact that I use `@sentry/browser` in my tests.
But since `@sentry/browser` (nor `@sentry/hub`) tests are not skipped in node 8. And the fact that you already have a bunch of tests using `getCurrentHub.withScope` in the hub package tells me that it's probably `jest` 🤔